webapp/src/widgets/icons/chevronDown.tsx

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react'
+
+import CompassIcon from './compassIcon'
+
+export default function ChevronDown(): JSX.Element {
```

Comment:
nit: I personally dislike this kind of icon names, I would prefer something more meaningful. Like the `Search` icon that you also included in this PR.

Reply:
It can be used for multiple purposes such as for dropdown indicators, scroll indicators, etc. The usage doesn't change the fact that it's just a generic, chevron down icon. I prefer to keep this `ChevronDown`.